hosts/voyager/iss.py

```diff
@@ -142,6 +142,25 @@ def from_file(filespec, astrometry=False, action='error', parameters={}):
                           label_dict['PRODUCT_ID'], RuntimeWarning)
             image_frame = spacecraft + '_ISS_' + camera
 
+    # Wrap the frame in a Navigation if necessary
```

Comment:
Is this idea intended to be applied to all hosts eventually?  If so, I'll create an issue.

Reply:
Yes, this is the idea. The other way to do it would be to have an Observation method called "navigate()", perhaps, that does this after the Observation was already created. That may be better because it would not have to be implemented for every host subclass. Regardless, we need an issue that says "figure out how to implement navigations of Observation objects".